### DISCUSSION TITLE
Jedis does not support returning PendingMessagesSummary

### DISCUSSION THREAD

**Discussion by jiangxiaoqiang** [xetorthio/jedis#2992] [Q&A] [answered]
when I using jedis 3.7.1 to fetch the redis pending stream message,shows error: Jedis does not support returning PendingMessagesSummary

more information: https://stackoverflow.com/questions/72195607/jedis-does-not-support-returning-pendingmessagessummary

Am I missing something?

**Comment by sazzad16** [comment, marked as answer]
You are not using Jedis directly. You are using it through spring-data-redis.

Even though Jedis has added the support of PendingMessagesSummary, spring-data-redis has not included the support of Jedis. That is why you are getting the error.

**Reply by sazzad16** [reply]
https://stackoverflow.com/a/72196043/4021802
